spmd/tensor/ops/pointwise_ops.py

```diff
@@ -142,6 +142,10 @@
     "aten.mul_.Tensor",
     "aten.ne.Scalar",
     "aten.tanh.default",
+    "aten.tanh_backward.default",
+    "aten.sgn.default",
+    "aten.neg.default",
+    "aten._softmax_backward_data.default",
```

Comment:
I guess we can resolve this later, but is `_softmax_backward_data` a point-wise op? My impression is that it's the same type as `softmax`.

Reply:
same question. Tried to apply the same rule as `softmax` to `softmax_backward` but failed on test.

Reply:
Regarding `_softmax_backward_data`, won't directly applying `pointwise_rule` miss the checking on shard dim vs. softmax dim?